src/components/common/YourCompany/BusinessStartDate.vue

```diff
@@ -39,24 +39,29 @@
           >
             <label class="start-date-title title-label">Start Date</label>
             <p class="mt-4">
-              Enter the start date of the business. The start date can be
-              <v-tooltip
-                top
-                max-width="20rem"
-                content-class="top-tooltip"
-                transition="fade-transition"
-              >
-                <template #activator="{ on }">
-                  <span
-                    class="tool-tip dotted-underline"
-                    v-on="on"
-                  >up to 2 years before the Registration Date</span>
-                </template>
-                <span>Choose the oldest date possible even if the actual start date is older than 2 years before
-                  the Registration Date.</span>
-              </v-tooltip>
-              and up 90 days after the Registration Date. Make certain that this is the correct date as it cannot be
-              easily corrected afterwards.
+              Enter the start date of the business.
+
+              <template v-if="!isRoleStaff">
+                The start date can be
+                <v-tooltip
+                  top
+                  max-width="20rem"
+                  content-class="top-tooltip"
+                  transition="fade-transition"
+                >
+                  <template #activator="{ on }">
+                    <span
+                      class="tool-tip dotted-underline"
+                      v-on="on"
+                    >up to 10 years before the Registration Date</span>
+                  </template>
+                  <span>Choose the oldest date possible even if the actual start date is older than 10 years before
+                    the Registration Date.</span>
+                </v-tooltip>
+                and up 90 days after the Registration Date.
+              </template>
```

Comment:
Excluded the above code for staff since they have no min start date rule and max start date is different for different filings.